lib/rules/rangeContextNodeParser.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+const rangeOperators = ['>=', '<=', '>', '<', '='];
+const styleSearch = require('style-search');
+
+function getRangeContextOperators(node) {
+	const operators = [];
+
+	styleSearch({ source: node.value, target: rangeOperators }, (match) => {
+		const before = node[match.startIndex - 1];
+
+		if (before === '>' || before === '<') {
+			return;
+		}
+
+		operators.push(match.target);
+	});
+
+	// Sorting helps when using the operators to split
+	// E.g. for "10em < width <= 50em" this returns ["<=", "<"]
+	return operators.sort((a, b) => b.length - a.length);
+}
+
+function getRangeContextName(parsedNode) {
+	if (parsedNode.length === 3) {
```

Comment:
Could you add a comment into the code explaining what is this `3`? It's not clear where this number came from.

Reply:
Yes, I added more context to that.